docs/tutorials/start-a-private-network/1compile.md

```diff
@@ -0,0 +1,25 @@
+---
```

Comment:
I will be more happy if the filename is `1_compile.md`.

Reply:
Took shawns advice to remove them. I do still prefer to keep them, and would be happy to type-set them this way.